examples/ping-pong/ping.go

```diff
@@ -25,14 +25,14 @@ func main() {
 	}
 	a, err := sctp.Client(config)
 	if err != nil {
-		log.Fatal(err)
+		log.Panic(err)
 	}
 	defer a.Close()
 	fmt.Println("created a client")
 
 	stream, err := a.OpenStream(0, sctp.PayloadTypeWebRTCString)
 	if err != nil {
-		log.Fatal(err)
+		log.Panic(err)
 	}
 	defer stream.Close()
 	fmt.Println("opened a stream")
```

Comment:
Got ```exitAfterDefer``` from ```gocritic```
I read about https://quasilyte.dev/blog/post/log-fatal-vs-log-panic/
Decided to change ```log.Fatal()``` to ```log.Panic()```